apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/connected-resources-panel.e2e.cy.ts

```diff
@@ -0,0 +1,33 @@
+describe(`open connected resources panel for resource`, () => {
+    const transcribedAudioBaseRoute = `/Resources/AudioItems/`;
+
+    const transcribedAudioId = `9b1deb4d-3b7d-4bad-9bdd-2b0d7b110110`;
+
+    describe(`when opening the connected resources panel for a resource`, () => {
+        beforeEach(() => {
+            cy.visit(`${transcribedAudioBaseRoute}${transcribedAudioId}`);
+        });
+
+        describe(`when the transcribed audio detail full view has loaded`, () => {
+            it(`should expose the open connected resources panel button`, () => {
+                cy.getByDataAttribute('HubIcon').should('exist');
```

Comment:
I don't like using data attributes that are named after visual design decisions. What if you want to change the icon? The point is you are getting a handle on the `connected-resource-panel-control` or `connected-resource-panel-expander` or something like that. Please update the data attribute for this one.